src/platform-includes/getting-started-config/javascript.mdx

```diff
@@ -13,7 +13,7 @@ Sentry.init({
   // Alternatively, use `process.env.npm_package_version` for a dynamic release version
   // if your build tool supports it.
   release: "my-project-name@2.3.12",
-  integrations: [new Sentry.BrowserTracing(), new Sentry.Replay()],
+  integrations: [new Sentry.BrowserTracing(), Sentry.replayIntegration()],
```

Comment:
Is this a bit weird to mix class/fn?

Reply:
`BrowserTracing` changes will be merged in soon, for now we just live with this, only will be like it for a week.